exasol_script_languages_container_tool/cli/commands/security_scan.py

```diff
@@ -57,16 +57,15 @@ def security_scan(flavor_path: Tuple[str, ...],
                                  target_docker_tag_prefix, "target")
 
     report_path = Path(output_directory).joinpath("security_scan")
-    task_creator = log_redirector_task_creator_wrapper(lambda: generate_root_task(task_class=SecurityScan,
-                                                                                  flavor_paths=list(flavor_path),
-                                                                                  report_path=str(report_path)
-                                                                                  ))
+    with log_redirector_task_creator_wrapper(lambda: generate_root_task(task_class=SecurityScan,
+                                                                        flavor_paths=list(flavor_path),
+                                                                        report_path=str(report_path)
+                                                                        )) as task_creator:
+        success, task = run_task(task_creator, workers, task_dependencies_dot_file)
 
-    success, task = run_task(task_creator, workers, task_dependencies_dot_file)
+        if success:
+            with task.security_report_target.open("r") as f:
+                print(f.read())
 
-    if success:
-        with task.security_report_target.open("r") as f:
-            print(f.read())
-    print(f'Full security scan report can be found at:{report_path}')
```

Comment:
Is this missing in the new version?

Reply:
The log for the report path has been moved into Tee.close()

Reply:
report path is not the log of exalsct, it is something different